define_variables_bionic.sh

```diff
@@ -12,6 +12,7 @@ export DYNAWO_JQUERY_DOWNLOAD_URL=http://localhost:8080
 export DYNAWO_FLOT_DOWNLOAD_URL=http://localhost:8080
 export DYNAWO_CPPLINT_DOWNLOAD_URL=http://localhost:8080
 export DYNAWO_XERCESC_DOWNLOAD_URL=http://localhost:8080
+export DYNAWO_PYTHON_COMMAND="python3"
```

Comment:
you need to add -e DYNAWO_PYTHON_COMMAND in dynawo_env below to get the variable in the container

Reply:
Solved